### DISCUSSION TITLE
v0.11.2

### DISCUSSION THREAD

**Discussion by gerwaric** [gerwaric/acquisition#63] [Announcements]
### Announcement

- I will be almost entirely OFFLINE from now until around **AUGUST 24, 2024**. I will be unable to work on acquisition until I return. Please report any new issues or updates, and I will review them as soon as I get back.

### Known Issues

There have been no code changes since [v0.11.2-rc.3](https://github.com/gerwaric/acquisition/releases/tag/v0.11.2-rc.3), but at least one new issue has been reported since release candidate 3.

- At least one user is experiencing an error while updating forums shops https://github.com/gerwaric/acquisition/issues/61

In addition, the following issues still seem to be occurring for some people:

- Some people may still be experiencing MSVCP140.dll related issues https://github.com/gerwaric/acquisition/issues/49
- Some HEAD request errors are still occurring https://github.com/gerwaric/acquisition/issues/60

### Help Needed (unchanged)

- I'm still seeing crash dumps related to HEAD reply network errors. Unless acquisition starts attaching logs files to the crash reports, I'm unable to see the details. If you are regularly experiencing this issue, please turn your logging to TRACE and report the last 5-10 lines from your log file leading up to the error here: https://github.com/gerwaric/acquisition/issues/60

### Bug Fixes (unchanged)

- I *think* this release may finally put an end to the recent MSVC-related crashes, which I'm still seeing in the crash reports (e.g. https://github.com/gerwaric/acquisition/issues/49). From what I can gather, these are due to incompatible versions of the MSVC dll files that cause memory access exceptions when using mutexes, which guard access to resources. I have removed the last remaining mutex, which I had added previously to make the datastores thread-safe. Now that I know more, I don't think they were necessary, and I'm not sure they solved the problem I intended them to solve.

TL;DR fingers crossed no more MSVC crashes with this release.

<hr /><em>This discussion was created from the release <a href='https://github.com/gerwaric/acquisition/releases/tag/v0.11.2'>v0.11.2</a>.</em>
